pkg/stm/stm_test.go

```diff
@@ -1,8 +1,12 @@
-package stm
+package stm_test
```

Comment:
package stm

Reply:
if use stm, it become cycle reference.

Reply:
import . "stm" ?

Reply:
metrics import stm and this test import metrics.